README.md

```diff
@@ -2,17 +2,17 @@
 
 <img align="right" width="200" src="https://cdn.jsdelivr.net/gh/YunYouJun/yun/images/yun-alpha-compressed.png">
 
-I am a learning master student from China, core team member of [element-plus](https://github.com/element-plus).
+I am a learning master student from China, and core team member of [element-plus](https://github.com/element-plus).
 I like open source and all interesting things and want to try to do it.
 
 I want to be an interesting person and create something that can be remembered by others.
 
-The one on the right is [Xiao Yun](https://github.com/YunYouJun/yun)（小云）, because YunYouJun is busy writing his graduation thesis, so she will introduce me to you.
+The one on the right is [Xiao Yun](https://github.com/YunYouJun/yun)（小云）, I'm busy writing my graduation thesis, so she will introduce me to you.
 
-- 🔭 I’m currently writing some amateur [open source project](https://github.com/YunYouJun?tab=repositories), interned at [@Tencent](https://github.com/Tencent/) + [@Alipay](https://github.com/alipay/).
+- 🔭 I’m currently writing some amateur [open source projects](https://github.com/YunYouJun?tab=repositories), interned at [@Tencent](https://github.com/Tencent/) + [@Alipay](https://github.com/alipay/).
 - 🌱 I’m currently learning Computer Graphics & Games & Drawing, and want to learn everything interesting.
-- 🤔 I want to make a pure front-end (more html & css than canvas) avg engine recently. I call it [advjs](https://www.advjs.org/).
-- ❤️ I like eating 🍉, raising 🐓, playing 🏓, sleeping in 🛌 and watching 📺 [ACGN](<https://en.wikipedia.org/wiki/ACG_(subculture)>).
```

Comment:
In fact, `<>` added by markdownlint, so i do not need to add `\_` for `_` in link. Can you revert this?